atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -768,14 +775,29 @@ private void cellsCleanedUp() {
                 line("@Override");
                 line("public boolean cellsCleanedUp(Transaction t, Set<Cell> cells) {"); {
                     line(StreamMetadataTable, " metaTable = tables.get", StreamMetadataTable, "(t);");
-                    line("Collection<", StreamMetadataRow, "> rows = Lists.newArrayListWithCapacity(cells.size());");
+                    line("Set<", StreamMetadataRow, "> rows = Sets.newHashSetWithExpectedSize(cells.size());");
```

Comment:
is it possible to add a comment in this file? not necessarily so that it renders as a comment in the generated code, but in this file itself?

wrangling with atlas types directly isn't inherently readable

Reply:
(As mentioned in overall review, renaming the variables would suffice)